lib/github_classroom/lti/request_signing.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module GitHubClassroom
```

Comment:
Are you able to write any tests for this Module? It may be possible by stubbing out the network requests.

Reply:
I've tried multiple approaches to writing tests for this module, and it seems that every time I'm essentially re-writing implementation of this mixin itself to test agains itself, which doesn't make too much sense.

No network requests are actually necessary so there's nothing to mock, as validating the signature only requires a consumer key, secret, request body, url, oauth_body_hash, and timestamp. 

Perhaps the quasi-integration tests of the services which rely on this mixin suffice?